engine/src/lin_time.cpp

```diff
@@ -144,7 +147,13 @@ void micro_sleep( unsigned int n )
 
 void micro_sleep( unsigned int n )
 {
-    (void) usleep( (useconds_t) n );
+    usleep(static_cast<useconds_t>(n));
+}
+
+#elif defined (__APPLE__) && defined (__MACH__)
```

Comment:
Probably better to look for all instances of micro_sleep and search and replace with sleep or nano_sleep.
https://stackoverflow.com/questions/42861913/why-use-usleep-and-not-sleep

Reply:
That's very useful info. I did not know about nano_sleep.

Maybe I can switch to that in a near-future PR?